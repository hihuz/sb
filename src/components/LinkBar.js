import React from "react";
import Link from "./Link";

const LinkBar = ({ links }) => (
  <div className="link-bar">
    {links.map(link => <Link {...link} />)}
  </div>
);

export default LinkBar;
