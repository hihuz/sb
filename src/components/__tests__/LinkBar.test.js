import React from "react";
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";
import LinkBar from "../LinkBar";

describe("LinkBar", () => {
  test("Snapshot", () => {
    const component = shallow(<LinkBar links={[]} />);
    const tree = shallowToJson(component);
    expect(tree).toMatchSnapshot();
  });
});
