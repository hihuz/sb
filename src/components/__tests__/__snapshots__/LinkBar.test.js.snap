// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LinkBar Snapshot 1`] = `
<div
  className="link-bar"
/>
`;
